api-public/src/main/kotlin/gropius/schema/mutation/ArchitectureMutations.kt

```diff
@@ -307,8 +307,7 @@ class ArchitectureMutations(
     }
 
     @GraphQLDescription(
-        """Creates a new Relation, requires RELATE_FROM_COMPONENT on the Component associated with 
-        start AND RELATE_TO_COMPONENT on the Component associated with end.
+        """Creates a new Relation, requires RELATE_FROM_COMPONENT on the Component associated with start.
```

Comment:
Maybe also add the required READ permission on the end side

Reply:
The idea is that READ is required on all Nodes defined via mutation inputs.
To not leak information, it should be transparent to the user whether a not does not exist, or the READ permission is missing.
Currently, this is not 100% implemented (we leak the information wether or not the node exists), however this will be fixed soon.
So, if you want this, this should be added for all id inputs for all mutation (I would prefer to add it to the general documentation of that service)

Reply:
Okay, makes sense to me. Thank you for the explanation